### PR TITLE
KF-85: As a CCX Coach, I want to message everyone enrolled in my course

### PR DESCRIPTION
**Pull request by MiroDojkic**

### CODE REVIEW

File: lms/djangoapps/ccx/urls.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 URLs for the CCX Feature.
 """
 from django.conf.urls import patterns, url
-
+from lms.djangoapps.ccx import views
```
Comment (mbareta) [inline lms/djangoapps/ccx/urls.py]: You don't need this.

File: lms/djangoapps/ccx/views.py [hunk 2]
```diff
@@ -687,3 +687,59 @@ def ccx_grades_csv(request, course, ccx=None):
...
+    ccx_message.save()
+
+    ccx_id = unicode(CCXLocator.from_course_locator(course.id, unicode(ccx.id)))
+
+    return redirect(reverse('ccx_messages', kwargs={'course_id': ccx_id}))
+
+@transaction.non_atomic_requests
+@coach_dashboard
+def ccx_messages_delete(request, course, ccx=None, **kwargs):
+    if 'message_id' in kwargs:
```
Comment (mbareta) [inline lms/djangoapps/ccx/views.py]: You should not write all this. Simple kwargs.get should suffice.

File: lms/templates/courseware/messages.html [hunk 3]
```diff
@@ -1,3 +1,4 @@
+## mako
```
Comment (mbareta) [inline lms/templates/courseware/messages.html:1]: What does this do?

File: lms/templates/ccx/ccx_messages_dashboard.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+        datePlaceholder = "{0:%Y}-{0:%m}-{0:%d}".format(datetime.datetime.date(datetime.datetime.now()))
+      %>
+      <div class="date-picker-wrapper">
+        <label for="date">Date:</label>
+        <input
+          id="date"
+          name="date"
+          class="date"
+          type="text"
```
Comment (mbareta) [inline lms/templates/ccx/ccx_messages_dashboard.html:66]: Add required attribute here.

File: lms/templates/courseware/messages.html (hunk 3 above)
Comment (MiroDojkic) [reply lms/templates/courseware/messages.html:1]: Ensures it's rendered as mako template

**Pull request merged**